2016-04-08.md

```diff
@@ -0,0 +1,37 @@
+We are thrilled to announce that Tessel 2 has shipped to all domestic and international pre-orders.
+
+To our knowledge, this is the first time a distributed, unpaid team of open source contributors has shipped a physical product. This is a major milestone in the open source hardware movement.
```

Comment:
Probably not unless you define this really narrowly. It's hard to make a clear distinction here with projects that became companies, projects that outlived companies, and open companies with external contributors, but I can think of several products that were developed collaboratively in a distributed OSHW community and shipped in quantity: ArduPilot, Milkymist, Ben NanoNote, HackRF, RepRap derivatives